docs/resources/server/loottables/index.md

```diff
@@ -0,0 +1,322 @@
+- The **loot context** is an object passed to the loot table evaluator that may contain additional information for rolling the table, for example an associated block state, block entity, entity, etc.; these are called **loot context parameters**.
+  - A full list of loot context parameters can be found in the `LootContextParams` class.
+- The **loot table type** is a validator for the loot context. For example, the loot table type `barter` is guaranteed to have an entity context for the piglin being bartered with.
+  - The context is validated at data pack load time, printing a log warning if context is used that is not defined by the loot table type.
+  - The loot table type is always optional. No validation will take place if it is not specified.
+  - This is internally called a **loot context parameter set**. The default parameter sets can be found in the `LootContextParamSets` class.
+  - For a full list of loot table types and what context they provide, see [Loot Contexts][lootcontext].
+- A **loot table evaluator** is the piece of Java code rolling the loot table.
+
+Generally, a loot table has the following JSON format:
+
+```json5
+{
+  "type": "chest", // loot table type
+  "neoforge:conditions": [
+    // data load conditions
+  ],
+  "functions": [
+    // table-wide loot functions
+  ],
+  "pools": [ // list of loot pools
+    {
+      "rolls": 1, // amount of rolls of the loot table, using 5 here will yield 5 results from the pool
+      "bonus_rolls": 0.5, // amount of bonus rolls, see the luck section below
+      "name": "my_pool",
+      "conditions": [
+        // pool-wide loot conditions
+      ],
+      "functions": [
+        // pool-wide loot functions
+      ],
+      "entries": [ // list of loot table entries
+        {
+          "type": "minecraft:item", // loot entry type
+          "name": "minecraft:dirt", // type-specific properties, for example the name of the item
+          "conditions": [
+            // entry-wide loot conditions
+          ],
+          "functions": [
+            // entry-wide loot functions
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+The exact [JSON specification of loot tables][loottablespec] can be found on the [Minecraft Wiki][wiki].
+
+## Using a Loot Table
+
+To roll a loot table, we need two things: the loot table itself, and a (possibly empty) set of loot context parameters.
+
+Let's start with getting the loot table itself. Loot tables are referenced by a [`ResourceLocation`][rl]. Loot table resource locations are relative to the `loot_tables` datapack folder. So for example, the loot table file for the loot table `examplemod:custom_fishing` would be located at `data/examplemod/loot_tables/custom_fishing`. Once you have the location, a loot table can then be obtained using `level.getServer().getLootData().getLootTable(location)`. As the loot data is only available through the server, this logic must run on a [logical server][sides], not a logical client.
```

Comment:
This does not exist. It's through `reloadableRegistries` and then `getLootTable`.